mflowgen/common/rtl/gen_rtl.sh

```diff
@@ -35,22 +37,19 @@ else
       echo "Use aha docker container for all dependencies"
 
       # Clone AHA repo
+      echo '--- gen_rtl aha clone BEGIN' `date +%H:%M`
       git clone https://github.com/StanfordAHA/aha.git
       cd aha
-      # install the aha wrapper script
-      pip install -e .
```

Comment:
Removing this "pip install" is the main focus of this pull. The rest is all fluff and refactor.